docs/generators/changelog/versions/2.13.md

```diff
@@ -26,12 +26,29 @@ TODO
 #### Meteor Version Release
 
 
+
+* `ddp-server@get-version`:
+    - Updated livedata server test to be more easily debbuged.
+
 * `Command line`:
```

Comment:
I like that we're naming this Command line instead of meteor-tool. Maybe it should have `meteor-tool` afterwards for consistency with the previous change logs (for example: `CLI (meteor-tool)`)?

Reply:
sounds nice!